code/modules/mapping/map_template.dm

```diff
@@ -77,6 +77,8 @@
 			continue
 
 		for(var/movable_in_turf in current_turf)
+			if(istype(movable_in_turf, /obj/docking_port/mobile))
```

Comment:
Maybe instead of doing this check for every object on the shuttle, we could just move the autosize stuff out of the Initialize call for the port and into its own proc that is explicitly called later? Doing it this way isn't the end of the world but it's probably best to avoid relying on the order things are initialized in if we have a cleaner way to do it.

Reply:
I'd have to make it where the xoffset, y offset stuff is saved onto the port itself, which would honestly just be var bloat given it will only be used once

Reply:
Alright fair enough then.